src/form/Validator.ts

```diff
@@ -29,7 +30,9 @@ export default class Validator {
 			return validationResult;
 		}
 
-		if ( !this.formValues.property || !this.formValues.value ) {
+		if ( !this.formValues.property ||
+			( !( this.formValues.propertyValueRelation === PropertyValueRelation.Regardless ) &&
```

Comment:
I feel this condition here is getting quite complex and hard to read. Is it possible to split it? maybe move some it into a new method. etc.

Reply:
I agree. I did struggle a bit with this. Do you have any suggestions on how to make it better?

Reply:
You can easily take out the property part (and define an incomplete form boolean and use that later to add the form error). I can add a follow up or something similar in a follow up PR.

Reply:
Also, there should be a test for this change to the Validator